react/react1/week2/todo-app/src/components/TodoItems.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from "react";
+
+function TodoItem({ item, todos, setTodo }) {
+  const [checked, setChecked] = useState(false);
```

Comment:
Would be much better to have this as the property of the todo item itself. Having this as a local state, it sets back to the default on every re-launch of the app. 
Try to add the ``completed`` property to the todo item and manage it that way 💡 👌🏽